auth-methods-styles.html

```diff
@@ -87,17 +87,15 @@
 
     .step-content .content {
       @apply(--layout-flex);
+      max-width: calc(100% - 16px);
+      @apply(--step-content);
     }
 
     paper-input,
     paper-masked-input {
       max-width: 560px;
     }
 
-    .step-content .content {
```

Comment:
This selector was duplicated.
You can compare styles [here](https://github.com/advanced-rest-client/auth-methods/blob/stage/auth-methods-styles.html#L88) and [here](https://github.com/advanced-rest-client/auth-methods/blob/feature/step-mixin/auth-methods-styles.html#L88).